generators/server/files/page.js

```diff
@@ -0,0 +1,25 @@
+import { createHash } from 'crypto';
+
+function torbit(markup) {
+  return createHash('sha256').update(markup).digest('hex');
+}
```

Comment:
Hmmm, it looks like Express already does something similar by default. http://stackoverflow.com/questions/24542959/how-does-a-etag-work-in-expressjs